easybuild/easyconfigs/m/MMseqs2/MMseqs2-11-e1a1c-iimpi-2019b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
```

Comment:
Please reinstate original attribution for this easyconfig. See `MMseqs2-10-6d92c-gompi-2019b`.